next_gen_ui_demo/lib/title_screen_3b/title_screen_ui.dart

```diff
@@ -0,0 +1,187 @@
+                onDifficultyFocused: onDifficultyFocused,
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _TitleText extends StatelessWidget {
+  const _TitleText();
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      mainAxisSize: MainAxisSize.min,
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        const Gap(20),
+        Row(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            Transform.translate(
+              offset: Offset(-(TextStyles.h1.letterSpacing! * .5), 0),
+              child: Text('OUTPOST', style: TextStyles.h1),
+            ),
+            Image.asset(AssetPaths.titleSelectedLeft, height: 65),
+            Text('57', style: TextStyles.h2),
+            Image.asset(AssetPaths.titleSelectedRight, height: 65),
+          ],
+        ),
+        Text('INTO THE UNKNOWN', style: TextStyles.h3),
+      ],
+    );
+  }
+}
+
+class _DifficultyBtns extends StatelessWidget {
+  const _DifficultyBtns({
+    required this.difficulty,
+    required this.onDifficultyPressed,
+    required this.onDifficultyFocused,
+  });
+
+  final int difficulty;
+  final void Function(int difficulty) onDifficultyPressed;
+  final void Function(int? difficulty) onDifficultyFocused;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      mainAxisSize: MainAxisSize.min,
+      children: [
+        _DifficultyBtn(
+          label: 'Casual',
+          selected: difficulty == 0,
+          onPressed: () => onDifficultyPressed(0),
+          onHover: (over) => onDifficultyFocused(over ? 0 : null),
+        ),
+        _DifficultyBtn(
+          label: 'Normal',
+          selected: difficulty == 1,
+          onPressed: () => onDifficultyPressed(1),
+          onHover: (over) => onDifficultyFocused(over ? 1 : null),
+        ),
+        _DifficultyBtn(
+          label: 'Hardcore',
+          selected: difficulty == 2,
+          onPressed: () => onDifficultyPressed(2),
+          onHover: (over) => onDifficultyFocused(over ? 2 : null),
+        ),
+        const Gap(20),
+      ],
+    );
+  }
+}
+
+class _DifficultyBtn extends StatelessWidget {
+  const _DifficultyBtn({
+    required this.selected,
+    required this.onPressed,
+    required this.onHover,
+    required this.label,
+  });
+  final String label;
+  final bool selected;
+  final VoidCallback onPressed;
+  final void Function(bool hasFocus) onHover;
+
+  @override
+  Widget build(BuildContext context) {
+    return FocusableControlBuilder(
+      onPressed: onPressed,
+      onHoverChanged: (_, state) => onHover.call(state.isHovered),
+      builder: (_, state) {
+        return Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: SizedBox(
+            width: 250,
+            height: 60,
+            child: Stack(
+              children: [
+                /// Bg with fill and outline
```

Comment:
This code is using doc comments in build methods, but it's more correct to use a normal comment since dartdoc won't generate anything anyway.

Reply:
done

Reply:
This is not the only place where there is a `///` in a build method...